include/openssl/core_names.h

```diff
@@ -42,8 +42,7 @@ extern "C" {
 
 /* digest parameters */
 #define OSSL_DIGEST_PARAM_XOFLEN    "xoflen"
-#define OSSL_DIGEST_PARAM_CMD       "cmd"
-#define OSSL_DIGEST_PARAM_MSG       "msg"
+#define OSSL_DIGEST_PARAM_SSL3_MS   "ssl3-ms"
```

Comment:
`ms` for "master secret", without being overly explicit.  Not that i know if that matters...  It makes me wonder, though, if there isn't another way to implement the concept (such as a Final with input...  could that be useful?)